Sources/CorePayments/Networking/GraphQL/GraphQLErrorResponse.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+/// Used to parse error message details out of GraphQL HTTP response body
+struct GraphQLErrorResponse: Decodable {
+    
+    let error: String
+    let correlationId: String?
```

Comment:
I think we want to do `ID` for the style guide. Not sure if that would require us to use a custom key though, but should be possible?
```suggestion
    let correlationID: String?
```

Reply:
It would require the custom keys, but it's possible! Addressed in 4ef956b78d8340d6eff9894965817d81e3be1482